src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -2,7 +2,13 @@
 
 # LocalVideoProvider
 
-The `LocalVideoProvider` provides the local video status, video tile id associated with the local user's video and a function to toggle their video.
+The `LocalVideoProvider` provides video tile id associated with the local user's video, the local video status, a function to set the value of `isVideoEnabled` and a function to toggle their video.
```

Comment:
Would it be helpful to say "toggle their local video" and also explain that the `isVideoEnabled` will provide status of the video tile rendered by the `LocalVideo` component. This is different to `PreviewVideo` component. But, since, we depend on the Amazon Chime SDK for JavaScript, a single video stream is attached to a video element at a time. Hence, closing preview video also affects closing of local video if enabled in the in-meeting screen page. Hence, we recommed using `PreviewVideo` component prior to in-meeting page.